ts_src/psbt.ts

```diff
@@ -129,6 +130,39 @@ export class Psbt {
     return this.data.inputs.length;
   }
 
+  get version(): number {
+    return this.__CACHE.__TX.version;
+  }
+
+  set version(version: number) {
+    this.setVersion(version);
+  }
+
+  get locktime(): number {
+    return this.__CACHE.__TX.locktime;
+  }
+
+  set locktime(locktime: number) {
+    this.setLocktime(locktime);
+  }
+
+  get txInputs(): TransactionInput[] {
+    return this.__CACHE.__TX.ins.map(input => ({
+      hash: cloneBuffer(input.hash),
+      index: input.index,
+      script: cloneBuffer(input.script),
```

Comment:
Don't include script. PSBT requires global transaction script to be empty, so this will always be a 0 length Buffer, showing it will encourage people to try and mess around with it.

Reply:
Done!